EIPS/eip-draft-whisper-packet-codes.md

```diff
@@ -0,0 +1,95 @@
+## Preamble
+
+    EIP: draft
+    Title: Whsiper Packet Codes - Specification
+    Author: Vlad Gluhovsky <gluk256@gmail.com>
+    Type: Informational
+    Status: Draft
+    Created: 2017-05-05
+
+## Abstract
+
+This draft EIP describes the packet codes and format used for Whisper messages within the ÐΞVp2p Wire Protocol.
+This EIP should substitute the [existing specification](https://github.com/ethereum/wiki/wiki/Whisper-Wire-Protocol).
+
+## Motivation
+
+It is necessary to specify the standard for Whipser messages in order to ensure forward compatibility of different Whisper clients, even in case they don't support particular message codes.
+
+## Specification
+
+All Whisper messages sent as ÐΞVp2p Wire Protocol packets should be RLP-encoded arrays of data containing two objects: integer packet code followed by another object (whose type depends on the packet code). 
+
+If Whisper node does not support a particular packet code, it should just ignore the packet without generating any error.
+
+### Packet Codes
+
+The Whisper sub-protocol should support the following packet codes:
+
+	Status (0x00)
```

Comment:
You should probably put these in a table with a column for EIP, so later EIPs can extend this set.